src/alerts/hipchat.coffee

```diff
@@ -0,0 +1,36 @@
+Alert = require './alert'
+request = require 'request'
+Hipchatter = require 'hipchatter'
+
+module.exports = class HipChatAlert extends Alert
+  constructor: (@config) ->
+    @api = new Hipchatter @config.key
+    # Check that the configured room exists, otherwise create it
+    @api.get_room @config.room, (err, response) ->
+      if err
+        # room does not exist, let's create it
+        room_data =
+          name: @config.room
+        @api.create_room room_data, (err, response) ->
+          if err
+            throw new Error "[Hipchat] Couldn't create room: #{err}"
+
+  sendToHipChat: (message) ->
+    options =
+      message: message
+      color: @config.color ? 'yellow'
```

Comment:
Using the `?` here means that a user can configure the color to be `''`. Will that cause an error, or does Hipchat/the API default to some standard color?